package.json

```diff
@@ -36,22 +37,27 @@
     "@actions/github": "^6.0.0",
     "@slack/web-api": "^7.5.0",
     "axios": "^1.7.7",
+    "axios-retry": "^4.5.0",
```

Comment:
Stoked to see retries being supported as it's a requested feature! But I was wondering if we would re-use web-api's provided retry feature? I haven't looked at the code yet, but the inclusion of this dependency tells me that we may not be using web-api's retry feature?

Reply:
🎉 I'm with you on this excitement! I'm also looking into running a few manual rate-limiting tests soon 😏

But I'm unsure if `axios-retry` can be replaced with the `webapi.retryPolicies` with how it's used right now... This is used when [sending requests using a webhook](https://github.com/slackapi/slack-github-action/blob/adf1dc21f8f74f1f81678eb25e43ef551e6f1e49/src/webhook.js#L84) with `axios` which seems to use somewhat different options, though IIRC `@slack/webapi` uses `axios` behind the scenes:

- `webapi.retryPolicies`: `retries`, `factor`, `randomize`
- `axios-retry`: `retries`, `retryDelay`, ...

I want this to work with `webapi.retryPolicies` though! Going to poke into this more in following experiments 👀